src/graphql-server/schema.graphql

```diff
@@ -66,6 +73,8 @@ type Transaction {
   presentBalance: Float
   notes: String
   happName: String
+  declinedReference: String
+  canceledReference: String
```

Comment:
How are we using these values in the front end? We only need to put stuff in graphql types that the frontend needs access to.